common/locales/en/subscriber.json

```diff
@@ -63,6 +63,30 @@
     "timeTravelersPopover": "We see you have a Mystic Hourglass, so we will happily travel back in time for you! Please choose the pet, mount, or Mystery Item Set you would like. You can see a list of the past item sets <a href='http://habitica.wikia.com/wiki/Mystery_Item' target='_blank'>here</a>! If those don't satisfy you, perhaps you'd be interested in one of our fashionably futuristic Steampunk Item Sets?",
     "timeTravelersAlreadyOwned": "Congratulations! You already own everything the Time Travelers currently offer. Thanks for supporting the site!",
     "mysticHourglassPopover": "A Mystic Hourglass allows you to purchase certain limited-time items, such as monthly Mystery Item Sets and awards from world bosses, from the past!",
+    "wingedMessengerSet":"Winged Messenger Set",
```

Comment:
I think these should be namespaced with the prefix "mysterySet" so you'd have:

```json
"mysterySetWingedMessenger": "Winged Messenger Set",
"mysterySetForestWalker": "Forest Walker Set",
// ...
```

You could even do:

```json
"mysterySet201402": "Winged Messenger Set",
"mysterySet201403": "Forest Walker Set",
// etc
```

And then you could remove the text attribute from the objects, and just loop over the mystery sets object (line 130-132) to set each text attribute to be:

```
// top of the file
let t = require('./translation.js');

// line 130
each(mysterySets, (value, key) => {
  value.key = key;
  value.text = t(`mysterySet${key}`);
});
```